sample-apps/react/react-dogfood/style/chat.scss

```
@@ -59,3 +59,77 @@
+  right: var(--str-video__spacing-lg);
+  bottom: 96px;
+  max-width: 280px;
+  padding: var(--str-video__spacing-md);
+  background: var(--str-video__background-color1);
+  border-radius: var(--str-video__border-radius-md);
+  box-shadow: 0 4px 16px rgba(0, 0, 0, 0.4);
+  z-index: 100;
+  animation: rd-chat__new-message-toast-in 0.18s ease-out;
+}
+
+@keyframes rd-chat__new-message-toast-in {
```

Stylelint (17.12.0)

[error] Expected keyframe name "rd-chat__new-message-toast-in" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)